Torch.Client/Manager/MultiplayerManagerLobby.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Torch.Client.Manager
 {
-    public class MultiplayerManagerLobby : MultiplayerManagerBase, IMultiplayerManagerServer
+    public class MultiplayerManagerLobby : MultiplayerManagerBase
```

Comment:
Leave it implementing IMultiplayerManagerServer.  Just make IsBanned always return false, and the ban list always return an empty list.